cmd/booster-http/server.go

```diff
@@ -36,6 +37,8 @@ var ErrNotFound = errors.New("not found")
 var lastModified = time.UnixMilli(1)
 
 const carSuffix = ".car"
+const piecePrefix = "piececid"
+const payloadPrefix = "payloadcid"
```

Comment:
I'd suggest using camel-case for query parameters, and renaming these variables to make clear they're parameters (not prefixes)
```suggestion
const pieceCidParam = "pieceCid"
const payloadCidParam = "payloadCid"
```